src/AppInstallerCLICore/Workflows/MultiQueryFlow.cpp

```diff
@@ -80,7 +80,7 @@ namespace AppInstaller::CLI::Workflow
             case SearchPurpose::Uninstall:

```

Comment:
Right above this we have two cases that basically decide whether to pass true or false to `SelectSinglePackageVersionForInstallOrUpgrade`, and then in that task we convert that bool to a `SearchPurpose`. Would it make sense to change `SelectSinglePackageVersionForInstallOrUpgrade` to also take a `SearchPurpose`?

Reply:
I think that makes sense, will update!